specifyweb/businessrules/views.py

```diff
@@ -113,7 +113,7 @@
     }
 })
 @login_maybe_required
-@require_http_methods(['GET', 'PUT'])
+@require_http_methods(['HEAD', 'GET', 'PUT'])
```

Comment:
(optional) could we create a wrapper for the `require_http_methods` decorator or some sort of middleware that makes every view that supports 'GET' also handle 'HEAD'?
there are no use cases for not supporting 'HEAD' when 'GET' is supported
having to specify it explicitly is only a source of bugs

Reply:
alternatively, we could make cache buster use GET instead of HEAD requests, but, HEAD are more efficient/faster for this purpose, and it's a good idea in general for API to support HEAD requests

Reply:
Perhaps not the best solution, but I could not explicitly find a way in native Django to accomplish what we're looking for. 
I've written the two decorators used to state that `GET` is required with our desired functionality and used these decorators throughout the backend: https://github.com/specify/specify7/pull/4435/commits/3498c8e6122f1a5c1778231f51c3e52acfb65796

It seems most places like those decorators are defined in `specify/views.py`, although I doubt that would be the best place for them: I would expect a `views.py` to only contain the views for the urls, and little to nothing else. 
I'd advocate for moving similar functions, classes, etc., outside of the `views.py` and into another spot (i.e., `middleware/general.py` if used extensively throughout the backend). 

Though I can move these two decorators to `specify/views.py` to maintain consistency.